### ISSUE TITLE
[iOS] Share to instagram story

### ISSUE THREAD

**Issue by tapkain** [esysberlin/esys-flutter-share#29]
I see that android can share to instagram story but iOS cant. Are there any plans to enhance this?

**Comment by devtronic** [comment]
I don't think that this is possible since the share targets depends on the target app itself.

**Comment by quetool** [comment]
Need this too. I think it is possible, here the documentation https://developers.facebook.com/docs/instagram/sharing-to-stories/

**Comment by ventskus-roman** [comment]
Strongly need this feature
